packages/lib/src/prod/federation_fn_import.js

```diff
@@ -32,15 +32,17 @@ async function getSharedFromRuntime(name, shareScope) {
     }
   }
   if (module) {
-    if (module.default) module = module.default
+    if (module?.default[Symbol.toStringTag] === 'Module')
```

Comment:
Perhaps we can refer to https://github.com/vuejs/core/blob/a95e612b252ae59eaf56e0b8ddba66948d4ac20e/packages/runtime-core/src/apiAsyncComponent.ts#L101 ?